sdk/ai/azopenai/MIGRATION.md

```diff
@@ -389,6 +389,304 @@ for _, embedding := range resp.Data {
 }
 ```
 
+### Legacy Completions
+
+**Before:**
+```go
+resp, err := client.GetCompletions(context.TODO(), azopenai.CompletionsOptions{
+    Prompt:         []string{"What is Azure OpenAI, in 20 words or less"},
+    MaxTokens:      to.Ptr(int32(2048)),
+    Temperature:    to.Ptr(float32(0.0)),
+    DeploymentName: to.Ptr("gpt-3.5-turbo-instruct"),
+}, nil)
+
+if err != nil {
+    // Handle error
+}
+
+for _, choice := range resp.Choices {
+    // Process each choice in the response
+    // *choice.Text() contains the generated text
+}
+```
+
+**After:**
+```go
+resp, err := client.Completions.New(context.TODO(), openai.CompletionNewParams{
+    Model: openai.CompletionNewParamsModel(model), // Azure deployment name here
+    Prompt: openai.CompletionNewParamsPromptUnion{
+        OfString: openai.String("What is Azure OpenAI, in 20 words or less"),
+    },
+    Temperature: openai.Float(0.0),
+})
+
+if err != nil {
+    // Handle error
+}
+
+for _, choice := range resp.Choices {
+    // Process each choice in the response
+    // choice.Text contains the generated text
+}
+```
+
+### Audio
+
+#### Transcription
+
+**Before:**
+```go
+resp, err := client.GetAudioTranscription(context.TODO(), azopenai.AudioTranscriptionOptions{
+    File: mp3Bytes,
+
+    ResponseFormat: to.Ptr(azopenai.AudioTranscriptionFormatText),
+
+    // DeploymentName: &modelDeploymentID,
+}, nil)
+
+if err != nil {
+    // Handle error
+}
+
+// Access response as *resp.Text
+
+```
+
+**After:**
+```go
+resp, err := client.Audio.Transcriptions.New(context.TODO(), openai.AudioTranscriptionNewParams{
+    Model:          openai.AudioModel(model), // Azure deployment name here
+    File:           audio_file, // Notice actual file object is passed here
```

Comment:
Very nice callout!

Reply:
Maybe for this (and the "before" bit), it would be helpful to show where the audio_file is coming to further illustrate that?